examples/scroll.py

```diff
@@ -151,25 +152,29 @@ def main(image_file=None):
         clock.tick()
 
         going = True
+
         while going:
             # wait for events before doing anything.
             # events = [pg.event.wait()] + pg.event.get()
             events = pg.event.get()
 
+            # During the loop, if a key is held, scroll the view.
+            keys = pg.key.get_pressed()
+            if keys[pg.K_UP]:
+                scroll_view(screen, image, DIR_UP, view_rect)
+            if keys[pg.K_DOWN]:
+                scroll_view(screen, image, DIR_DOWN, view_rect)
+            if keys[pg.K_LEFT]:
+                scroll_view(screen, image, DIR_LEFT, view_rect)
+            if keys[pg.K_RIGHT]:
+                scroll_view(screen, image, DIR_RIGHT, view_rect)
```

Comment:
This change in code here, means that this example does not need to set keyrepeat anymore (because those are not handled)

Reply:
I think we should merge this and then perhaps make a few mental notes (or issues) to fiddle with later. Original PR submitter has waited long enough, and we could probably fiddle with the examples forever (and will).